### PR TITLE
Rename daemon-user-count and make an alias

### PR DESCRIPTION
**Pull request by Hoverbear**
We discussed that this may make the UX more consistent.

Now:

```
❯ cargo run -- install --help | rg nix-
   Compiling nix-installer v0.0.0-unreleased (/home/ana/git/determinatesystems/harmonic)
    Finished dev [unoptimized + debuginfo] target(s) in 9.23s
     Running `target/debug/nix-installer install --help`
To pass custom options, select a planner, for example `nix-installer install linux-multi --help`
Usage: nix-installer install [OPTIONS] [PLAN]
       nix-installer install <COMMAND>
      --nix-build-user-count <NIX_BUILD_USER_COUNT>
      --nix-build-group-name <NIX_BUILD_GROUP_NAME>
      --nix-build-group-id <NIX_BUILD_GROUP_ID>
      --nix-build-user-prefix <NIX_BUILD_USER_PREFIX>
      --nix-build-user-id-base <NIX_BUILD_USER_ID_BASE>
      --nix-package-url <NIX_PACKAGE_URL>
          [default: https://releases.nixos.org/nix/nix-2.12.0/nix-2.12.0-x86_64-linux.tar.xz]
          If `nix-installer` should forcibly recreate files it finds existing

```

### CODE REVIEW

**Review by cole-h — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "nix-installer"
 description = "A `nix` installer"
-version = "0.0.0-unreleased"
+version = "0.0.1-unreleased"
```
[the remark this answers is not in the document]
Comment (cole-h) [reply Cargo.toml]: lol you beat me to it

**Comment by Hoverbear** [comment]
@cole-h sorry I rebased this and also realized I forgot to change one of the actions.

File: tests/fixtures/darwin/darwin-multi.json [hunk 2]
```diff
@@ -1,687 +1,687 @@
 {
-  "version": "0.0.0-unreleased",
```
[the remark this answers is not in the document]
Comment (Hoverbear) [reply tests/fixtures/darwin/darwin-multi.json:2]: Ah-ha! I see how do to it!!!!

File: src/settings.rs [hunk 3]
```diff
@@ -289,8 +290,8 @@ impl CommonSettings {
 // Builder Pattern
 impl CommonSettings {
     /// Number of build users to create
-    pub fn daemon_user_count(&mut self, count: usize) -> &mut Self {
-        self.daemon_user_count = count;
+    pub fn nix_user_count(&mut self, count: usize) -> &mut Self {
```
Comment (cole-h) [inline src/settings.rs:293]: Do we want to rename this to `nix_build_user_count`?

File: src/settings.rs [hunk 4]
```diff
@@ -256,8 +257,8 @@ impl CommonSettings {
             serde_json::to_value(modify_profile)?,
         );
         map.insert(
-            "daemon_user_count".into(),
-            serde_json::to_value(daemon_user_count)?,
+            "nix_user_count".into(),
```
Comment (cole-h) [inline src/settings.rs:260]: Same here.

File: src/settings.rs [hunk 5]
```diff
@@ -231,7 +232,7 @@ impl CommonSettings {
         let Self {
             channels,
             modify_profile,
-            daemon_user_count,
+            nix_build_user_count: nix_user_count,
```
Comment (cole-h) [inline src/settings.rs:235]: And here.

**Review by cole-h — APPROVED** [review]
Still LGTM, a few minor questions I missed last time though:

File: src/settings.rs (hunk 3 above)
Comment (Hoverbear) [reply src/settings.rs:293]: I think so!

**Pull request merged**